app/livechat/server/api/lib/visitors.ts

```diff
@@ -120,12 +155,18 @@ export async function searchChats({
 	return {
 		history,
 		count: history.length,
-		offset,
-		total: (total && total.count) || 0,
+		offset: pagination.offset,
+		total: total?.count || 0,
 	};
 }
 
-export async function findVisitorsToAutocomplete({ userId, selector }) {
+export async function findVisitorsToAutocomplete({
+	userId,
+	selector,
+}: {
+	userId: string;
+	selector: any | string;
```

Comment:
Here, `selector` is not `any`. From line 173 we see that it's an object with 2 array keys `exceptions` and `conditions`. You can type these both keys as `any[]` 

```
selector: { exceptions: any[]; conditions: any[] }
```

Reply:
thanks for pointing that out!